index.js

```diff
@@ -17,13 +14,10 @@ function getDown (defaults) {
   }
 }
 
-function getLocation (name) {
-  mkdirp.sync(tmpdir)
-  return path.join(tmpdir, name)
+function getLocation () {
```

Comment:
We should document that the `name` argument (https://github.com/Level/level-test#db--ctorname-options-callback) isn't always used. Or maybe remove `name` altogether?

Reply:
I think it's better if we remove it altogether.

Reply:
Agreed